warp/src/crypto/cipher.rs

```diff
@@ -1,6 +1,5 @@
 #![allow(clippy::result_large_err)]
 #[allow(unused)]
-#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Does `wasm32-unknown-unknown` support `io::Read` and `io::Write` now? We could probably move `std::io::ErrorKind` out into its own import (since I do know this would error if this attr is used for `ErrorKind`) and keep this so we are not including `Read` or `Write` it by mistake unless it is supported by wasm32 arch.

Reply:
that said I separated ErrorKind as you suggested